pim/utils.py

```diff
@@ -113,17 +123,52 @@ def remove(self, name):
 
     def show(self):
         if len(self.required) == 0:
-            echo('\nNo packages installed')
+            echo('\nNo requirements.')
         else:
-            echo('\nInstalled packages: ')
-            with indent(4, quote='  -'):
-                for package in self.required:
-                    puts(package)
+            echo('\nCurrent requirements: ')
+            for package in self.required:
+                echo(' - ' + str(package))
```

Comment:
ah yes, i was actually wanting to switch to `click.echo` everywhere, except it doesn't work alongside `clint.indent` e.g. https://github.com/freeman-lab/pim/blob/master/pim/commands/init.py#L36-L42